PowerEditor/src/NppDarkMode.cpp

```diff
@@ -456,24 +506,30 @@ namespace NppDarkMode
 	COLORREF getDarkerBackgroundColor()   { return getTheme()._colors.pureBackground; }
 	COLORREF getErrorBackgroundColor()    { return getTheme()._colors.errorBackground; }
 	COLORREF getTextColor()               { return getTheme()._colors.text; }
+	COLORREF getHotTextColor()            { return lightColor(getTheme()._colors.text, TEXTLUMINANCE_MAX); }
 	COLORREF getDarkerTextColor()         { return getTheme()._colors.darkerText; }
 	COLORREF getDisabledTextColor()       { return getTheme()._colors.disabledText; }
 	COLORREF getLinkTextColor()           { return getTheme()._colors.linkText; }
 	COLORREF getEdgeColor()               { return getTheme()._colors.edge; }
 	COLORREF getHotEdgeColor()            { return getTheme()._colors.hotEdge; }
 
-	HBRUSH getBackgroundBrush()           { return getTheme()._brushes.background; }
-	HBRUSH getSofterBackgroundBrush()     { return getTheme()._brushes.softerBackground; }
-	HBRUSH getHotBackgroundBrush()        { return getTheme()._brushes.hotBackground; }
-	HBRUSH getDarkerBackgroundBrush()     { return getTheme()._brushes.pureBackground; }
-	HBRUSH getErrorBackgroundBrush()      { return getTheme()._brushes.errorBackground; }
-
-	HBRUSH getEdgeBrush()                 { return getTheme()._brushes.edgeBrush; }
-	HBRUSH getHotEdgeBrush()              { return getTheme()._brushes.hotEdgeBrush; }
+	HBRUSH getBackgroundBrush()          { return getTheme()._brushes.background; }
+	HBRUSH getSofterBackgroundBrush()    { return getTheme()._brushes.softerBackground; }
+	HBRUSH getHotBackgroundBrush()       { return getTheme()._brushes.hotBackground; }
+	HBRUSH getDarkerBackgroundBrush()    { return getTheme()._brushes.pureBackground; }
+	HBRUSH getErrorBackgroundBrush()     { return getTheme()._brushes.errorBackground; }
```

Comment:
The 5 above added lines are identical as the 5 removed lines, please restore them.